target/linux/lantiq/image/vr9.mk

```diff
@@ -127,6 +127,16 @@ define Device/avm_fritz7360sl
 endef
 TARGET_DEVICES += avm_fritz7360sl
 
+define Device/avm_fritz7360v2
+  $(Device/AVM)
+  DEVICE_MODEL := FRITZ!Box 7360
+  DEVICE_VARIANT := v2
+  IMAGE_SIZE := 32128k
+  DEVICE_PACKAGES := kmod-ath9k kmod-owl-loader wpad-basic kmod-usb-dwc2
+  SUPPORTED_DEVICES += FRITZ7360V2
```

Comment:
Is that required for some reason? I do not see a reason, so I'd remove it with a separate patch.

Despite, I'd like to introduce a hyphen before the v2 in the device node name/compatible ...